parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -252,8 +232,6 @@ write_files:
     KUBELET_REGISTER_SCHEDULABLE=true
     KUBELET_NODE_LABELS={{GetAgentKubernetesLabels . "',variables('labelResourceGroup'),'"}}
 
-AGENT_ARTIFACTS_CONFIG_PLACEHOLDER
```

Comment:
This implementation vector has been officially retired in favor of the standard `{{CloudInitData "<file reference>"}}` usage pattern. See `/etc/systemd/system/kubelet.service` and `/etc/systemd/system/kms.service` above.